<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="del1-lhp-n65294" ignored="0" name="Test" tests="4" failures="1" timestamp="08 Aug 2019 07:08:13 GMT" time="46.676" errors="0">
  <testcase name="test1" time="10.822" classname="multi.threads.NewTest"/>
  <testcase name="test2" time="15.512" classname="multi.threads.NewTest"/>
  <testcase name="test3" time="5.82" classname="multi.threads.NewTest"/>
  <testcase name="stop_all" time="30.008" classname="multi.threads.NewTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at multi.threads.NewTest.stop_all(NewTest.java:42)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- stop_all -->
</testsuite> <!-- Test -->
